monitor/getBalances.js

```diff
@@ -103,6 +105,18 @@ async function main(bridgeMode) {
       logger.debug('Methods for half duplex token are not present')
     }
 
+    try {
+      logger.debug('calling foreignBridge.methods.chaiToken')
+      const chaiToken = await foreignBridge.methods.chaiToken().call()
+      if (chaiToken !== ZERO_ADDRESS) {
```

Comment:
does it make sense to also check `isChaiTokenEnabled`?

Reply:
I agree I think we should use `isChaiTokenEnabled` instead of `chaiToken` since the address will be defined after migrating the contracts but could not be enabled yet.